style.css

```diff
@@ -0,0 +1,10 @@
+body{
+    padding: 0;
+    margin: 0;
+  }
+  #main{
+    background: silver;
+    text-align: center;
+    
+  }
```

Comment:
Kindly avoid adding Unnecessary blank lines and  spaces  as this makes your code unclean